plugins/parcel-transformer-markdown-html/lib/index.js

```diff
@@ -155,6 +155,7 @@ module.exports = new Transformer({
 			const tocFilename = config.contents.tocPath
 				? path.join(workingDir, config.contents.tocPath)
 				: null;
+			let fileToImport, importFileParsedCode;
```

Comment:
I think most of the changes in this PR (including standard.html) can be reverted in favor of this terribly hard to read fancy oneliner:

```
const parsedCode = attributes.import
            ? await options.inputFS
                    .readFile(path.join(process.cwd(), attributes.import), 'utf-8')
                    .then(contents => marked(contents))
            : marked(body);
```

Reply:
Indeed, updated